packages/babel-helper-create-class-features-plugin/src/index.ts

```diff
@@ -168,8 +168,8 @@ export function createClassFeaturePlugin({
 
         if (!props.length && !isDecorated) return;
 
+        const innerBinding = path.node.id;
         let ref;
-
         if (path.isClassExpression() || !path.node.id) {
```

Comment:
can reuse tne innerBinding and line 177?
```suggestion
        if (path.isClassExpression() || !innerBinding) {
```